src/Raven.Server/Documents/ETL/EtlTransformer.cs

```diff
@@ -377,6 +379,11 @@ private JsValue HasTimeSeries(JsValue self, JsValue[] args)
             return false;
         }
         
+        private JsValue GetRevisionsCount(JsValue self, JsValue[] args) => 
+            (Current.Document.Flags & DocumentFlags.HasRevisions) == DocumentFlags.HasRevisions 
```

Comment:
You use `Current` here while according to the signature you might pass any document (not only `this`).

I think the following will not work:

```

var doc = load('users/1');

var count = getRevisionsCount(doc);

```

Reply:
From what I can see, the methods in `EtlTransformer.cs` do not accept additional arguments for analyzing different documents. It seems that we should maintain this convention in our code

Reply:
see here: 
https://github.com/ravendb/ravendb/blob/a83d5a4a968c32319807c3103c53bc2866a01dee/src/Raven.Server/Documents/ETL/EtlTransformer.cs#L231